src/components/UI/Charts/Sunburst/Sunburst.tsx

```diff
@@ -120,7 +128,11 @@ class Sunburst extends Component<SunburstProps, State> {
           ref={this.ref}
         />
         <div className={'grid-item'}>
-          <InfoPanel data={{ ...selectedPhenotypeInfo }} treeData={phenotypeTree} />
+          <InfoPanel
+            data={{ ...selectedPhenotypeInfo }}
```

Comment:
I don't think that copying here is needed since props are copied (to be verified)